packages/api-symphony-compatibility/src/mapping.ts

```diff
@@ -0,0 +1,56 @@
+const containerjsSsf = ssf;
+
+namespace map {
+  export class ssf {
+    static activate(windowName: string) {
+      // Could use window.focus in containerjs, but no way of selecting window by name yet
```

Comment:
OK, fair enough - that does look like a bit of an omission in our current API - and is something covered in #171